README.md

```diff
@@ -97,192 +62,117 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **Technician endpoint**
+- **Users can add/remove a Technician**
+- **Users can book a Technician**
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+## 💻 Website Mockup <a name="screenshots"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## Entity Relationship Diagram <a name="er-diagram"></a>
+<img width="850" alt="erd" src="https://user-images.githubusercontent.com/86082070/217819998-c8acf559-359c-4654-99e1-84027a0e6d05.jpeg">
 
-<!-- LIVE DEMO -->
+<!-- Kanban Board -->
+<!-- API Docs -->
+<!-- ER DIAGRAM-->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo]()
+
+## [API Documentation](https://experthub-production.up.railway.app/api-docs/index.html) <a name="api-documentation"></a>
+
+<img width="715" alt="api-docs" src="https://user-images.githubusercontent.com/86082070/217821271-c90e1a90-e81d-4ed2-b4ce-08e9d960a7e4.png">
+
+<img width="703" alt="api-docs" src="https://user-images.githubusercontent.com/86082070/217821444-053f5958-6983-4105-a562-2860e98cebf7.png">
+
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
-
 ### Prerequisites
 
-In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
+Make sure you have `Git` and `Ruby` installed
 ```
- -->
-
-### Setup
-
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+ gem install rails
 ```
---->
 
 ### Install
```

Comment:
- [ ] Kindly, include instructions for the db commands `rails db:create, rails db:migrate, rails db:seed` that your users will require to properly run the project. 
- [ ] You are not pushing to the repo the master key.  In that case, kindly include the instructions for your users to generate one: 

```markdown
Remove config/master.key and config/credentials.yml.enc
Run in terminal:

EDITOR=code rails credentials:edit 

This command will create a new master.key and credentials.yml.enc if they do not exist.
```

That way, your users will be able to properly run the project. 👍